sql/pgwire_test.go

```diff
@@ -446,7 +461,7 @@ func TestPGPreparedQuery(t *testing.T) {
 		}
 	}
 
-	initStmt := `CREATE DATABASE d; CREATE TABLE d.t (a INT); INSERT INTO d.t VALUES (10),(11); CREATE TABLE d.ts (a TIMESTAMP, b DATE);`
+	initStmt := `CREATE DATABASE d; CREATE TABLE d.t (a INT); INSERT INTO d.t VALUES (10),(11); CREATE TABLE d.ts (a TIMESTAMP, b DATE); CREATE TABLE d.tsd (a TIMESTAMP DEFAULT NOW(), b INT);`
```

Comment:
No reason for this to be one long line. Let's add some newlines after every statement for readability.